keanu-python/tests/test_base.py

```diff
@@ -15,34 +15,28 @@ def __init__(self, numbers):
 
             super(JavaListWrapper, self).__init__(lst)
 
-        def index_of(self, value):
+        def get(self, index):
             return 100
 
     return JavaListWrapper([1, 2, 3])
 
 
-def test_java_object_wrapper_cant_call_java_api_with_no_python_impl_if_camel_case(java_list_wrapper) -> None:
-    with pytest.raises(AttributeError) as excinfo:
-        java_list_wrapper.isEmpty()
+def test_unwrapped_can_call_java_api(java_list_wrapper) -> None:
+    assert not java_list_wrapper.unwrap().isEmpty()
 
-    assert str(excinfo.value) == "{} has no attribute isEmpty".format(type(java_list_wrapper))
 
+def test_wrapped_can_call_python_api(java_list_wrapper) -> None:
+    assert java_list_wrapper.get(0) == 100
 
-def test_java_object_wrapper_cant_call_java_api_with_python_impl_if_camel_case(java_list_wrapper) -> None:
-    with pytest.raises(AttributeError) as excinfo:
-        java_list_wrapper.indexOf(1)
 
-    assert str(excinfo.value) == "{} has no attribute indexOf. Did you mean index_of?".format(type(java_list_wrapper))
+def test_throws_if_there_is_no_java_api(java_list_wrapper) -> None:
+    with pytest.raises(AttributeError, match="{} has no attribute missingMethod".format(type(java_list_wrapper))):
+        java_list_wrapper.missingMethod()
```

Comment:
this test would be more impactful if instead of `missingMethod` you were to use a method that does exist on the unwrapped object, e.g. `java_list_wrapper.isEmpty()`

Reply:
And can you add a similar test for `java_list_wrapper.is_empty()`